src/components/product-menu-dropdown/product-menu-dropdown.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from '@mapbox/react-icon';
+
+class ProductMenuDropdown extends React.PureComponent {
+  render() {
+    const categoryLength = this.props.categories.length;
+    const activeDotStyle = {
+      content: '',
+      width: '6px',
+      height: '6px',
+      borderRadius: '3px',
+      background: '#4264fb',
+      position: 'absolute',
+      margin: '10px 0 0 -12px',
+      display: 'none'
+    };
```

Comment:
Since this object doesn't get affected by the props, it could be declared outside the component's `render` method, which means we don't have to re-create the object on every render.